apps/dashboard/app/apps/router.rb

```diff
@@ -30,7 +30,6 @@ def self.router_from_token(token)
   def self.pinned_apps(tokens, all_apps)
     @pinned_apps ||= {}
     tokens_key = ActiveSupport::Cache.expand_cache_key(tokens)
-    return @pinned_apps[tokens_key] if @pinned_apps.key?(tokens_key)
```

Comment:
I know caching is a bit all over the place, but was this added by accident?

Reply:
Yes, this was committed by accident. Reverting....